app/src/main/java/org/cru/godtools/fragment/ToolsFragment.java

```diff
@@ -210,6 +242,17 @@ private boolean showDownloading() {
         return mMode == MODE_AVAILABLE;
     }
 
+    private void closeTutorial() {
+        mSettings.setFeatureDiscovered(Settings.FEATURE_OPT_IN_TUTORIAL);
+        mToolsBinding.setIsTutorialViewable(false);
```

Comment:
this should be unnecessary once you correctly monitor for changes to the feature flag